cocos/input/types/event-enum.ts

```diff
@@ -265,7 +265,7 @@ export enum SystemEventType {
      * @en The event type for node's sibling order changed.
      * @zh 当节点在兄弟节点中的顺序发生变化时触发的事件。
      *
-     * @deprecated since v3.3, please use Node.EventType.SIBLING_ORDER_CHANGED instead
+     * @deprecated since v3.3, please use Node.EventType.CHILDREN_ORDER_CHANGED instead
```

Comment:
Not since 3.3. Only 3.8.2 has this API.

Reply:
`SystemEventType. SIBLING_ORDER_CHANGED` is deprecated since v3.3.